src/Hocon.Tests.Performance/ConfigLookupSpec.cs

```diff
@@ -26,6 +26,7 @@ public class ConfigLookupSpec
         public void Setup(BenchmarkContext context)
         {
             _throughputCounter = context.GetCounter(ConfigLookupName);
+            System.GC.Collect();
```

Comment:
Probably don't need this - NBench does this automatically.

Reply:
Not the last one